src/dex-ui/pages/dao/DAODashboard.tsx

```diff
@@ -0,0 +1,181 @@
+import { Box, Center, Flex, Tab, TabList, Tabs } from "@chakra-ui/react";
+import { ErrorLayout, LoadingSpinnerLayout, NotFound, Page, PageLayout } from "@layouts";
+import {
+  Color,
+  Notification,
+  NotficationTypes,
+  useNotification,
+  LayoutIcon,
+  TransactionIcon,
+  BoxIcon,
+  LockIcon2,
+  UsersIcon,
+  SettingsIcon,
+} from "@dex-ui-components";
+import { useTabFilters } from "@hooks";
+import { useLocation, NavLink, useNavigate, useParams } from "react-router-dom";
+import { PropsWithChildren } from "react";
+import { DAO } from "@services";
+import { Paths } from "@dex-ui/routes";
```

Comment:
You can directly access alias paths without the `@dex-ui/` portion (e.g. `import { Paths } from "@routes";`)

Reply:
Changes the import path.